src/osdc/Striper.cc

```diff
@@ -524,13 +524,14 @@ uint64_t Striper::StripedReadResult::assemble_result(
     bufferlist *bl)
 {
   ldout(cct, 10) << "assemble_result(" << this << ")" << dendl;
+  uint64_t off = 0;
```

Comment:
Please ignore the updates. I think I need to remap it in https://github.com/ceph/ceph/blob/master/src/librbd/io/ReadResult.cc#L162